example/function/src/functions.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
You will also need to update the instructions in the README for creating and invoking the function from the example

Reply:
Sure. I would be glad to do this :)

Reply:
@clareliguori If you take a look at lines 120-122 in the README, I have updated the invocation example as only that is what changes. Everything else remains the same.

Reply:
Oh duh, silly me, it's right there ;)

Reply:
Oh wait this is what I was referring to: `zip hello.zip src/hello.php`

Reply:
Oh Shoot! I have now fixed that and another typo in the create-function example. Since we have done a cd in function directory, output directory is 2 levels and not 1 level up. :)